scripts/py_matter_idl/matter_idl/tests/outputs/proto/ota_software_update_provider_cluster.proto

```diff
@@ -0,0 +1,107 @@
+
+/// AUTO-GENERATED with matter_idl.
+
+syntax = "proto3";
+
+package com.matter.example.proto;
+
+option java_multiple_files = true;
+
+message OtaSoftwareUpdateProviderCluster {
+  // option (message_type) = MATTER_TRAIT;
+
+  enum ClusterId {
+    CLUSTER_ID_UNSUPPORTED = 0;
+    CLUSTER_ID = 41;
+  }
+
+  // Enums
+  enum OTAApplyUpdateAction {
+    OTAAUA_PROCEED = 0;
+    OTAAUA_AWAIT_NEXT_ACTION = 1;
+    OTAAUA_DISCONTINUE = 2;
+  }
+
+  enum OTADownloadProtocol {
+    OTADP_BDXSYNCHRONOUS = 0;
+    OTADP_BDXASYNCHRONOUS = 1;
+    OTADP_HTTPS = 2;
+    OTADP_VENDOR_SPECIFIC = 3;
+  }
+
+  enum OTAQueryStatus {
+    OTAQS_UPDATE_AVAILABLE = 0;
+    OTAQS_BUSY = 1;
+    OTAQS_NOT_AVAILABLE = 2;
+    OTAQS_DOWNLOAD_PROTOCOL_NOT_SUPPORTED = 3;
+  }
+
+
+
+
+  // Structs
```

Comment:
bug? Different filtering for comment vs loops?

Reply:
I guess I was arguing for not having the comments at all and using the potential bugs as an extra justification. I am claiming that having `//enums` for example does not help me because:
   - in the template this preceeds a for-loop that obivously iterates over enums
   - in the proto files, this comment preceeds several `enums` so the purpose is obvious
   - it seems easy to make bugs and add extra spacing

So please remove these comments. They use up vertical space, are buggy and do not help reading of the code.

Reply:
What I like about having the section comments, is when you generate a lot of artifacts, you can deterministically see for a given cluster whether it has structs or not.  For example, a trivial scan of all the test output right now shows that the custom generator has no coverage currently for Struct output.